server/data/nomisClientBuilder.js

```diff
@@ -107,7 +107,7 @@ function nomisGetBuilder(token) {
             throw new NoTokenError();
         }
 
-        logger.info(`Calling nomis with token: ${token}`);
+        logger.debug(`Calling nomis with token: ${token}`);
 
```

Comment:
Maybe don't need this now we're sorted with tokens. May create noise in logs. Up to you